e2e/__snapshots__/templates.test.js.snap

```diff
@@ -5748,8 +5767,11 @@ function App() {
         <InstantSearch searchClient={searchClient} indexName=\\"indexName\\">
           <div className=\\"search-panel\\">
             <div className=\\"search-panel__filters\\">
-              <RefinementList attribute=\\"facet1\\" />
-              <RefinementList attribute=\\"facet2\\" />
+              <Configure facets={['*']} maxValuesPerFacet={20} />
```

Comment:
Is this still valid, given it's going to be a different default in a near future?

Reply:
Yes, and this also isn't relevant to this PR @tkrugg, it was added in #534. When we release a version which supports facets * + maxValuesPerFacet out of the box, I'll change the template and the lowest allowed version